tests/testthat/test-fixed_regex_linter.R

```diff
@@ -40,6 +40,7 @@ test_that("fixed_regex_linter blocks simple disallowed usages", {
   expect_lint("regexec('\\\\$', x)", lint_msg, linter)
   expect_lint("grep('\n', x)", lint_msg, linter)
   expect_lint("grep('\\\\;', x)", lint_msg, linter)
+  expect_lint("grep('\\\\/', x)", lint_msg, linter)
```

Comment:
maybe we need a patrick test for all the characters, wdyt?

Reply:
SGTM. Will add them.

Reply:
I have added parametrized tests now.